### DISCUSSION TITLE
SQL snippets not being shared within a team in SQL Editor

### DISCUSSION THREAD

**Discussion by clonecorp** [supabase/supabase#7040] [Feature Requests]
Hello,

I'd like to know if the SQL snippets that we create in the SQL Editor can be shared between members of a team within the same project. Other developers can't see the SQL snippets I wrote and I cannot see theirs but this is mandatory to debug functions or anything SQL-related.

Is this something you can fix?

Thank you,
Jordan.

**Comment by MildTomato** [comment]
Moving to discussions

**Comment by MaybeThisIsRu** [comment]
Our team would like to see this too.

**Comment by ameesme** [comment]
It's a bit confusing that this isn't default behavior as any other panel in the UI _is_ shared among team-members.

**Comment by lizastrong** [comment]
any update?

**Comment by smartfox87** [comment]
Our team would like to see this too.

**Comment by poseiso** [comment]
I need this fr fr

**Comment by foufrix** [comment]
Same here, this is not the behavior expected as we use it in code using `supabase.rpc("nameFunction", {args})`

Do we need to create an issue for this one?

**Comment by carloseam94** [comment]
I would really appreciate having this option too. To be honest, it seems like it should be the default behavior to begin with.

**Comment by CareTiger** [comment]
+1. we need this too

**Comment by saltcod** [comment]
Hey all — we shipped this feature last Launch Week. Thanks for the idea!

![screenshot-2023-10-06-at-15 50 07](https://github.com/supabase/supabase/assets/105593/590f3e24-cfaa-4066-8bbd-8c8df4417c65)

**Reply by geeniusdope** [reply]
OK, this helps, but there is still an issue. Dev A cant edit code built by Dev B. This is not great. When something is "shared", that should mean shared for collaboration, not just duplication.
